docker-compose.yml

```diff
@@ -1,18 +1,25 @@
-version: "3.7"
+version: "3"
+
+volumes:
+  bundle:
 
 services:
-  meilisearch_ruby:
-    image: ruby
+  package:
```

Comment:
A way to make the docker service more user-friendly.

But it could be `lib`, `integration`, `sdk`. WDYT @alallema and @bidoubiwa ?

Reply:
I really like the idea to add one docker-compose in different SDK.

Reply:
@brunoocasali yes I understand! I didn't answer this question. It was just a comment.
But on my `docker-compose` I use `sdk`, not sure it is the best term maybe package or integration is better.

Reply:
Oh, I see, I thought you answered it ahaha, yeah I liked the package as well :)